server/app.js

```diff
@@ -18,7 +26,18 @@ const routes = require("./routes");
 const { routeNotFound, errorHandler } = require("./middlewares");
 
 const app = express();
-app.use(morgan("tiny"));
+
+app.use(
```

Comment:
remove morgan dependency since we are going to use winston. If possible.

Reply:
Removed use of morgan. Only using Winston now for transporting logs.

Reply:
Remove the morgan dependency itself. If it's not required.

Reply:
Removed morgan dependency completely.